nixos/modules/services/networking/zerotierone.nix

```diff
@@ -7,6 +7,16 @@ let
 in
 {
   options.services.zerotierone.enable = mkEnableOption "ZeroTierOne";
+
+  options.services.zerotierone.joinNetwork = mkOption {
+    default = null;
```

Comment:
can you make this an array? It's possible to join multiple networks at the same time.

Reply:
done.